Sources/JWTKit/JWTKeyCollection.swift

```diff
@@ -38,14 +47,14 @@ public actor JWTKeyCollection: Sendable {
     func add(_ signer: JWTSigner, for kid: JWKIdentifier? = nil) -> Self {
         let signer = JWTSigner(algorithm: signer.algorithm, parser: signer.parser, serializer: signer.serializer)
 
-        if let kid = kid {
+        if let kid {
             if self.storage[kid] != nil {
-                print("Warning: Overwriting existing JWT signer for key identifier '\(kid)'.")
+                logger.warning("Overwriting existing JWT signer", metadata: ["kid": "\(kid)"])
```

Comment:
These should be debug as per the log level guide https://www.swift.org/documentation/server/guides/libraries/log-levels.html